Update browser list to match vault page

leo-shopify opened the pull request:
- Dropping IE support
- Last two versions of Safari and IOS
- Update Android to 6; Latest is 8

Browser support [vault page](https://vault.shopify.com/Browser-Support#admin-and-1st-party-apps-and-channels)

Commit:
Update browser list to match vault page

- Dropping IE support
- Last two versions of Safari and IOS
- Update Android to 6; Latest is 8

GoodForOneFare commented on browsers.js:
```diff
@@ -2,9 +2,7 @@ module.exports = [
   'last 3 chrome versions',
   'last 3 firefox versions',
   'last 2 versions',
```
"last 2 versions" gets IE11 back into the list.  [Live example](http://browserl.ist/?q=last+3+chrome+versions%2C+last+3+firefox+versions%2C+last+2+versions%2C+safari+%3E%3D+10%2C+ios+%3E%3D+10%2C+android+%3E%3D+6).  

There's [an open PR to prune IE](https://github.com/Shopify/babel-preset-shopify/pull/20), but I haven't had time to dig into @lemonmade's questions.

Commit:
Constraint list exclude defaults

- Explicitly add last 3 version of Edge
- Replace android (web view) with Chrome for android

This gets support to just under 72% of [global coverage]

[global coverage]: http://browserl.ist/?q=last+3+chrome+versions%2C+last+3+firefox+versions%2C+last+3+chromeandroid+versions%2C+last+2+edge+versions%2C+safari+%3E%3D+10%2C+ios+%3E%3D+10

leo-shopify commented:
This PR is too specific to admin's. The [other related PR](https://github.com/Shopify/babel-preset-shopify/pull/20/files) offers wider support.
